client/client_auth_test.go

```diff
@@ -25,20 +25,6 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
-type fakeAuthorizer struct {
-	rsp       []byte
-	rspErr    error
-	url       string
-	reqCalled bool
-}
-
-func (f *fakeAuthorizer) Request(api ApiRequester, url string, adm AuthDataMessenger) ([]byte, error) {
-	fmt.Printf("url: %s\n", url)
-	f.url = url
-	f.reqCalled = true
-	return f.rsp, f.rspErr
-}
```

Comment:
I'd leave these two separate. The changes happen across 2 distinct commits that don't relate with each other and touch different parts of the tree. IMO it's not worth the hassle.